modules/ui/commit_warnings.js

```diff
@@ -34,7 +37,7 @@ export function uiCommitWarnings(context) {
 
             containerEnter
                 .append('h3')
-                .call(severity === 'warning' ? t.append('commit.warnings') : t.append('commit.errors'));
+                .call(t.append(`commit.${severity}s`));
```

Comment:
One advantage of using the https://github.com/openstreetmap/iD/pull/11020/files#diff-677e75df5348a6b4af7e5becaec89f67be3bae43dcf3b2b6a9535bd5781dfa5eR91-R96 in for translations as well is those translations strings are easier to find with global search.